libs/elements/layout/header/src/lib/app-header.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HeaderComponent } from './components/header/header.component';
+
+@NgModule({
+  imports: [CommonModule],
+  declarations: [HeaderComponent],
+  exports: [HeaderComponent]
+})
+export class AppHeaderModule {}
```

Comment:
Good job on this file, just add a space between angular imports and local imports